docs/_documentations/private-registries.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title: Developing with packages from private registries and repositories 
+description: Developing with packages from private registries and repositories 
+keywords: install, installing, IntelliJ, Eclipse, Codewind, IDE, plugin, plug-in, settings, creating, project, projects, template, code change, edit, edits, application, removing, private, registry, registries, repository, repositories, package, package 
+duration: 5 minutes 
+permalink: private-registries  
+type: document 
+---
+
+# Developing with packages from private registries and repositories 
+
+Packages, reusable pieces of software, help you build and reduce development time. Sometimes, the packages for your application must be installed from an external private registry or one hosted by your organization. Registries help you manage your code and dependencies. Codewind supports two private registry types: NPM registries for Node.js projects and Maven repositories for Java projects. 
+
+## Developing with packages from private NPM registries 
+
+1. Configure your local system to access the private registry. To do this, add the registry configuration and credentials to your profile's `.npmrc` file. For more information, refer to the [NPM documentation](https://docs.npmjs.com/configuring-npm/npmrc.html). 
+    **Note:** you should not put the `.npmrc` file inside your project as it contains sensitive data.
+2. Add a [reference](referencing-files.html) to the `.npmrc` file in your project.
+3. Install packages from the configured registry to your project locally (e.g. `npm install @myorg/mypackage`).Once Codewind builds the Docker image for your applicaiton, it can pull down the packages from the private registry you have configured.
+
+## Developing with packages from private Maven repositories 
+
+1. Configure your local system to access the private repository. To do this, add the repository server credentials to your profile's `settings.xml` file. For more information, refer to the [Maven documentation](https://maven.apache.org/settings.html#Servers). 
+    **Note:** you should not put the `settings.xml` file inside your project as it contains sensitive data.
+2. Add a [reference](referencing-files.html) to the `settings.xml` file in your project.
+3. Modify your `pom.xml` file, then add the appropriate configuratioin to your `<repositories>`/`<pluginRepositories>`, then add your `<dependencies>` element references to the packages your application requires. Once Codewind builds the Docker image for your application, it can pull down the packages from the private repository you have configured.
```

Comment:
Should it be "... add **to** your `<dependencies>` element..."?  The `<dependencies>` element contains a list of `<dependency>` elements, and we're add to that list

Reply:
Okay, I thought we were adding the ```dependency elements``` to the ```packages your application requires```. 

It's actually, adding the ```packages your application requires``` to the ```dependency elements```, right?

Reply:
We are adding a `<dependency>` to each of the package we want to use

```
<dependencies>
    <dependency>
      <groupId>com.ibm.navigator</groupId>
      <artifactId>navigatorAPI</artifactId>
      <version>1.0</version>
    </dependency>
    ...
</dependencies>
```